### PR TITLE
ci: Add broken links check

### PR DESCRIPTION
**Pull request by matevz**

### CODE REVIEW

File: .github/workflows/blc.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Broken link checker
```
Comment (tjanez) [inline .github/workflows/blc.yml]: Can you rename the file to `ci-links.yml`?

Comment (tjanez) [inline .github/workflows/blc.yml]: Also, can you add `name` fields to steps and comment them similar to https://github.com/oasisprotocol/oasis-core/blob/master/.github/workflows/ci-lint.yml?

**Review by tjanez — COMMENTED** [review]
Can you group the changes into 3 commits:
- Add ci-links.yml
- Change config
- Fix actual broken links

**Comment by tjanez** [comment]
Also, can you set it up so it is also run daily besides for PRs?

**Comment by matevz** [comment]
I noticed the CI is occasionally getting error 429 from github because there are too many requests. Let's try to limit those.

**Comment by matevz** [comment]
> I noticed the CI is occasionally getting error 429 from github because there are too many requests. Let's try to limit those.

Testing [linkinator](https://github.com/JustinBeckwith/linkinator) which supports multiple retries and multiple levels of verbosity.

**Comment by matevz** [comment]
Tested linkinator and it has some nice advantages over broken-link-checker:
- retries on errors like 429 or if the site is not yet up (e.g. when running `yarn serve`) with increasing timeouts
- supports different verbosity levels (useful for reporting broken links only)
- supports config file (no need to pass everything as command line in `packages.json`)

As far as I tested it locally and with github actions, it is more robust than broken-link-checker and should be sufficient for the cron jobs.

**Review by tjanez — COMMENTED** [review]
Can you split the last commit "ci: Add ci-links.yml" into 2 parts:
- Adding ci-lint.yml
- Changing broken links checker to linkinator

Otherwise, looks good!

**Review by tjanez — APPROVED** [review]

**Pull request merged**